hudi-client/hudi-client-common/src/main/java/org/apache/hudi/metrics/HoodieMetrics.java

```diff
@@ -175,6 +175,7 @@ public void updateMetricsForEmptyData(String actionType) {
     Metrics.registerGauge(getMetricsName(actionType, "totalCompactedRecordsUpdated"), 0);
     Metrics.registerGauge(getMetricsName(actionType, "totalLogFilesCompacted"), 0);
     Metrics.registerGauge(getMetricsName(actionType, "totalLogFilesSize"), 0);
+    Metrics.registerGauge(getMetricsName(actionType, "totalRecordsDeleted"), 0);
```

Comment:
/nit would prefer to have this line moved up after "totalInsertRecordsWritten" so similar metrics grouped together